api/src/glfs-resolve.c

```diff
@@ -395,6 +396,7 @@ glfs_resolve_component(struct glfs *fs, xlator_t *subvol, inode_t *parent,
             errno = ENOMEM;
             goto out;
         }
+        ret = dict_set_int32_sizen(xattr_req, GF_NAMESPACE_KEY, 1);
```

Comment:
I think we can error out like in 395 on failure?

Reply:
Ack, will do.